### PR TITLE
Create map pin

### LINKED ISSUE #877: East Lansing
## Adding East Lansing to the map

Hi everyone! This issue is to track the addition of East Lansing to the map, which can be found at:

- Longitude: -84.483865
- Latitude: 42.736979

### PR DESCRIPTION
**Pull request by fricanol**
Add username, longitude/latitude coordinates. The training team is @githubteacher. Closes #877

### CODE REVIEW

File: _pins/fricanol.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: fricanol
+latitude: 42.7370° N
+longitude: 84.4839° W
```
Comment (jr-cologne) [inline _pins/fricanol.json]: Please remove `° N` and `° W`. Then it should work.

Comment (hectorsector) [inline _pins/fricanol.json]: @fricanol let me know if there's anything I can do to help.

**Comment by isema** [comment]
Update branch and merge pull request

**Comment by nitincgit** [comment]
pls add me

**Pull request merged**